cadvisor.go

```diff
@@ -62,6 +62,8 @@ var collectorKey = flag.String("collector_key", "", "Key for the collector's cer
 
 var storeContainerLabels = flag.Bool("store_container_labels", true, "convert container labels and environment variables into labels on prometheus metrics for each container. If flag set to false, then only metrics exported are container name, first alias, and image name")
 
+var basePrefix = flag.String("base_prefix", "", "prefix path that will be prepended to all paths to support some reverse proxies")
```

Comment:
can we make this less generic-sounding?  Maybe `url_base_prefix`?

Reply:
Good idea.